### PR TITLE
Bugfix/dapp broken transaction history on small screens

### PR DESCRIPTION
**Pull request by weilbith**
**Thank you for submitting this pull request :)**

Fixes part of [this comment](https://github.com/raiden-network/light-client/issues/2604#issuecomment-822344517) in #2604

**Short description**

gscreenshot_2021-05-21-163411

**Definition of Done**

- [x] Steps to manually test the change have been documented
- [x] Acceptance criteria are met
- [ ] Change has been manually tested by the reviewer (dApp) 

**Steps to manually test the change (dApp)**

1. See it yourself on `320x568`

### CODE REVIEW

File: raiden-dapp/src/components/AmountDisplay.vue [hunk 1]
```diff
@@ -24,10 +24,16 @@ import type { Token } from '@/model/types';
 export default class AmountDisplay extends Vue {
   @Prop({ required: false, default: false, type: Boolean })
   exactAmount!: boolean;
+
   @Prop({ required: true })
   amount!: string | BigNumber;
+
+  @Prop({ type: String, default: '' })
```
Comment (taleldayekh) [inline raiden-dapp/src/components/AmountDisplay.vue:31]: I don't think you need to have that empty string default value?

**Review by taleldayekh — APPROVED** [review]
Nice, thanks 👍🏽

File: raiden-dapp/src/components/AmountDisplay.vue (hunk 1 above)
Comment (weilbith) [reply raiden-dapp/src/components/AmountDisplay.vue:31]: I do. Else the rendering will throw an error as we do not render it condiationally.

**Comment by weilbith** [comment]
#2727 has been merged and thereby the end-to-end tests do work again. I'll rebase now.

**Comment by weilbith** [comment]
I'm confused. The CI workflow fails for the linting. Locally it works. So I re-run the CI job with SSH. I compared the `git rev-parse HEAD` commit hashes to be equal. I used `sha1sum yarn.lock` to make sure they are the same as well. Then I force re-installed all dependencies with `yarn install --force --forzen-lockfile` on both machines. But the linting still fails in CI and succeeds on local. The issue it is complaining doesn't seem to make sense.

**Comment by weilbith** [comment]
Okay I don't care why the linting has different results. I know understood the warning and fixed it. Optimistically I'll integrate the fixup commit already to save CI time.

**Pull request merged**